minio/resource_minio_iam_user_policy_attachment.go

```diff
@@ -55,9 +55,10 @@ func minioCreateUserPolicyAttachment(ctx context.Context, d *schema.ResourceData
 func minioReadUserPolicyAttachment(ctx context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
 	minioAdmin := meta.(*S3MinioClient).S3Admin
 	var userName = d.Get("user_name").(string)
+	var isLDAPUser = LDAPUserDistinguishedNamePattern.MatchString(userName)
 
 	userInfo, errUser := minioAdmin.GetUserInfo(ctx, userName)
-	if errUser != nil {
+	if errUser != nil && !isLDAPUser {
```

Comment:
I guess that would be exactly the same as for groups - if it will always fail on LDAP groups, it might be better to not execute `GetUserInfo` if minio can't get information about LDAP users.

Reply:
Like for group, this is completely valid in MinIO, as you can see here:

```
$ mc admin user info ttc "CN=Me,OU=User,DC=my,DC=org"
AccessKey: CN=Me,OU=User,DC=my,DC=org
Status: 
PolicyName: consoleAdmin
MemberOf: 
$ mc admin user info ttc "CN=MissingUser,OU=User,DC=my,DC=org"
mc: <ERROR> Unable to get user info. The specified user does not exist. (Specified user does not exist).
```